tests/events/evthandler.cpp

```diff
@@ -122,6 +122,9 @@ class MyClassWithEventTable : public wxEvtHandler
     void OnEvent(wxEvent&) { g_called.method = true; }
     void OnAnotherEvent(AnotherEvent&);
     void OnIdle(wxIdleEvent&) { g_called.method = true; }
+#ifdef wxHAS_NOEXCEPT
+    void OnIdleNoExcept(wxIdleEvent&) noexcept { }
```

Comment:
Why do you need `wxHAS_NOEXCEPT`. Can't you just do
 `void OnIdleNoExcept(wxIdleEvent&) wxNOEXCEPT { }`

Reply:
I could do it like this, but this would silently do nothing if `wxNOEXCEPT` was defined as nothing, and it seemed slightly preferable to use `noexcept` explicitly in the case when it's supposed to be available.

Of course, this still has the same problem with `wxHAS_NOEXCEPT` not being defined when it should be, so it's not really a solution neither...

To be honest, I've already hesitated for a long time between the two and I'm not sure if it's worth spending any more time on this.